samples/tls/logging/README.md

```diff
@@ -19,35 +19,28 @@ you run any of the scripts in this repository:
 Generating these certificates is outside the scope of this example. However, you
 can use the process documented in ["Configure NGINX Ingress TLS for SAS Applications"](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=default&docsetId=calencryptmotion&docsetTarget=n1xdqv1sezyrahn17erzcunxwix9.htm&locale=en#n0oo2yu8440vmzn19g6xhx4kfbrq) in SAS Viya Administration documentation and specify the `logging` namespace.
 
-## Instructions
+## Using This Sample
```

Comment:
TLS is always required and always enabled for the logging components.  The one exception is HTTPS access to Kibana.  Bryan and I need to talk through this sample to see if it is needed/correct.

Reply:
As we discussed earlier today, this sample's README needs revisions to address other issues and that work will be done in a separate effort.